### ISSUE TITLE
[bug] javacpp -pytorch  centos7 java23 meet  jvm crash

### ISSUE THREAD

**Issue by mullerhai** [bytedeco/javacpp-presets#1598]
Hi , on centos7  run  lstm layer  to predict ,meet jvm crash, maybe the libjnitorch  has something  unsuite jdk version ?

jdk 23
scala 3.6
sbt 1.20.6
centos 7
gcc 9
lib
glibc 2.31
use cpu

log

(base) [root@ddp2 storch_demo2]# /usr/local/jdk-23.0.1/bin/java  -jar ./target/scala-3.6.2/storch_demo-assembly-0.1.0-SNAPSHOT.jar
Warning: Version of org.bytedeco:pytorch could not be found.
Warning: Version of org.bytedeco:openblas could not be found.
Using device: Device(CPU,-1)
model LSTM(inputSize=28, hiddenSize=128,numLayers=2,batchFirst = true,dropout = 0.1,bidirectional = false bias=true)
Linear(inFeatures=128, outFeatures=10, bias=true)
model LstmNet(
  (lstm): LSTM(inputSize=28, hiddenSize=128,numLayers=2,batchFirst = true,dropout = 0.1,bidirectional = false bias=true)
  (fc): Linear(inFeatures=128, outFeatures=10, bias=true)
)
ds.size.get {ds.size.get} data_loader option 32
coming in for loop
Epoch: 1 | Batch:  200 | Training loss: 1.1247 | Eval loss: 1.1797 | Eval accuracy: 0.5616
Epoch: 1 | Batch:  400 | Training loss: 1.5196 | Eval loss: 1.0088 | Eval accuracy: 0.6189
Epoch: 1 | Batch:  600 | Training loss: 0.6277 | Eval loss: 0.8607 | Eval accuracy: 0.6807
Epoch: 1 | Batch:  800 | Training loss: 0.7155 | Eval loss: 0.7952 | Eval accuracy: 0.7028
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007faac26b71c7, pid=71882, tid=71883
#
# JRE version: Java(TM) SE Runtime Environment (23.0.1+11) (build 23.0.1+11-39)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (23.0.1+11-39, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, linux-amd64)
# Problematic frame:
# C  [libjnitorch.so+0x14871c7]
#
# Core dump will be written. Default location: /opt/software/storch_demo2/core.71882
#
# An error report file with more information is saved as:
# /opt/software/storch_demo2/hs_err_pid71882.log
[27.119s][warning][os] Loading hsdis library failed
#
# If you would like to submit a bug report, please visit:
#   https://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.

**Comment by mullerhai** [comment]
I  have test cpu  and  gpu on centos7 jdk 23, all get error jvm crash

**Comment by mullerhai** [comment]
^C(base) [root@ddp2 storch_demo2]# /usr/local/jdk-23.0.1/bin/java  -jar ./target/scala-3.6.2/storch_demo-assembly-0.1.0-SNAPSHOT.jar
Warning: Version of org.bytedeco:pytorch could not be found.
Warning: Version of org.bytedeco:openblas could not be found.
Using device: Device(CUDA,-1)
model LSTM(inputSize=28, hiddenSize=128,numLayers=2,batchFirst = true,dropout = 0.1,bidirectional = false bias=true)
Linear(inFeatures=128, outFeatures=10, bias=true)
model LstmNet(
  (lstm): LSTM(inputSize=28, hiddenSize=128,numLayers=2,batchFirst = true,dropout = 0.1,bidirectional = false bias=true)
  (fc): Linear(inFeatures=128, outFeatures=10, bias=true)
)
ds.size.get {ds.size.get} data_loader option 32
coming in for loop
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fc2f70b212c, pid=119312, tid=119313
#
# JRE version: Java(TM) SE Runtime Environment (23.0.1+11) (build 23.0.1+11-39)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (23.0.1+11-39, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, linux-amd64)
# Problematic frame:
# C  [libstdc++.so.6+0xb112c]  std::__exception_ptr::exception_ptr::_M_addref()+0xc
#
# Core dump will be written. Default location: /opt/software/storch_demo2/core.119312
#
# An error report file with more information is saved as:
# /opt/software/storch_demo2/hs_err_pid119312.log
#
# If you would like to submit a bug report, please visit:
#   https://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

**Comment by saudet** [comment]
Duplicate of https://github.com/bytedeco/javacpp-presets/issues/1379
